tests/features/subscriptions.feature

```diff
@@ -0,0 +1,58 @@
+@api
+Feature: Subscription
+  In order to manage my subscriptions on the website
```

Comment:
Maybe clarify the intent of the user a bit better, he does not want to manage his subscriptions on the website, instead he wants to be notified about changes to certain content on the website.